prover/src/pp.rs

```diff
@@ -28,28 +31,31 @@ where
     Ok(())
 }
 
-pub fn load_pp<SP>(file: &str) -> Result<PP<SP>, ProofError>
+pub fn load_pp<C, SP>(file: &str) -> Result<PP<C, SP>, ProofError>
 where
+    C: CommitmentScheme<P1>,
     SC: StepCircuit<F1> + Sync,
-    SP: SetupParams<G1, G2, C1, C2, RO, SC> + Sync,
+    SP: SetupParams<G1, G2, C, C2, RO, SC> + Sync,
 {
     let f = File::open(file)?;
     let mut dec = Decoder::new(&f)?;
-    let pp = PP::<SP>::deserialize_compressed(&mut dec)?;
+    let pp = PP::<C, SP>::deserialize_compressed_unchecked(&mut dec)?;
```

Comment:
Why is this change needed?

Reply:
I thought it wouldn't make a difference anymore due to the arkworks update: is
this not true?